testing/src/cypress/page-objects/Institution-objects/DashboardInstitutionObject.ts

```diff
@@ -55,31 +71,19 @@ export default class DashboardInstitutionObject {
     return cy.contains("PROFILE");
   }
 
-  profileIconButton() {
-    return cy.get(".mdi-account");
-  }
-
-  logOffButton() {
-    return cy.contains("Log off");
-  }
-
-  confirmationEnrollmentText() {
-    return cy.contains("Confirm enrollment ");
-  }
-
-  confirmationEnrollmentVerifyText() {
-    return cy.contains("Available to confirm enrollment");
+  iconButton() {
+    return cy.get(".fa-user");
```

Comment:
we can use data-cy to get this button.

Reply:
will start data-cy labels to the code

Reply:
Is the data-cy part committed? I couldn't see here.